packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -117,13 +117,25 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   private handlers: ChatHandlers;
   private emitter: EventEmitter = new EventEmitter();
 
-  constructor(chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient) {
+  constructor(chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient, options?: ChatAdapterOptions) {
     this.bindAllPublicMethods();
     this.chatClient = chatClient;
     this.chatThreadClient = chatThreadClient;
     this.context = new ChatContext(chatClient.getState(), chatThreadClient.threadId);
-    /* @conditional-compile-remove(file-sharing) */
-    this.fileUploadAdapter = new AzureCommunicationFileUploadAdapter(this.context);
+    const credential = options?.credential;
+    if (credential) {
+      /* @conditional-compile-remove(teams-inline-images) */
+      const getAuthToken: () => Promise<string> = async () => {
+        const accessToken = await credential.getToken();
+        return accessToken.token;
+      };
+      /* @conditional-compile-remove(teams-inline-images) */
+      this.fileUploadAdapter = new AzureCommunicationFileUploadAdapter(this.context, getAuthToken);
+    } else {
+      /* @conditional-compile-remove(file-sharing) */
```

Comment:
We can do 
let getAuthToken = undefined;
/* @conditional-compile-remove(teams-inline-images) */
getAuthToken: () => Promise<string> = async () => {}

this.fileUploadAdapter = new AzureCommunicationFileUploadAdapter(this.context, getAuthToken);

So we can save an if-else in CC and less code to clean up